pkg/executables/govc.go

```diff
@@ -270,13 +270,15 @@ func (g *Govc) DeployTemplateFromLibrary(ctx context.Context, templateDir, templ
 		}
 	}
 
-	logger.V(4).Info("Taking template snapshot", "templateName", templateName)
-	if err := g.createVMSnapshot(ctx, templateName); err != nil {
+	templateFullPath := filepath.Join("/", templateDir, templateName)
```

Comment:
`templateDir` already includes the `/` in the front, so we shouldn't need to add it here manually
https://github.com/aws/eks-anywhere/blob/3992ec57de953137f10ebcb8360dd2768d19e1db/pkg/providers/vsphere/vsphere.go#L859

Reply:
templateDir being passed into DeployTemplateFromLibrary is just the dir

https://github.com/aws/eks-anywhere/blob/3992ec57de953137f10ebcb8360dd2768d19e1db/pkg/executables/govc.go#L244